Kickstarter-iOS/Views/Cells/SettingsPrivacyDeleteAccountCell.swift

```diff
@@ -0,0 +1,63 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+import UIKit
+
+public protocol SettingsPrivacyDeleteAccountCellDelegate: class {
+  func goToDeleteAccount(url: URL)
```

Comment:
How about:

`func settingsPrivacyDeleteAccountCellTappedDelete(_ cell: SettingsPrivacyDeleteAccountCell)`

Reply:
Renamed it to `func settingsPrivacyDeleteAccountCellTapped(_ cell: SettingsPrivacyDeleteAccountCell, with url: URL)` but let me know if there's a better name for it!

Reply:
I think that's good! 👍